mne/viz/evoked.py

```diff
@@ -1819,18 +1912,39 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
             head_pos = {'center': (0, 0), 'scale': (0.5, 0.5)}
             pos, outlines = _check_outlines(pos, np.array([1, 1]), head_pos)
             if not isinstance(show_sensors, (np.int, bool)):
-                raise TypeError("`show_sensors` must be numeric or bool, not" +
-                                str(type(show_sensors)))
+                raise ValueError("show_sensors must be numeric or bool, not" +
```

Comment:
isn't `TypeError` more suitable here?

Reply:
I always get that confused. But I think this is a ValueError because it's the value of a kwarg.

Reply:
I don't think Type/Value usually has to do with whether it was positional or keyword, but rather the type of error (so in this case I agree `TypeError` seems more suitable)